tests/test_edge_cases.py

```diff
@@ -144,24 +164,26 @@ async def test_stage_checksum_missmatch(
     # place the content for an example file in the permanent storage:
     file_object = file_fixture.copy(
         update={
-            "bucket_id": joint_fixture.config.staging_bucket,
-            "object_id": EXAMPLE_FILE.file_id,
+            "bucket_id": joint_fixture.staging_bucket,
+            "object_id": EXAMPLE_METADATA.object_id,
         }
     )
     await joint_fixture.s3.populate_file_objects(file_objects=[file_object])
 
     # populate the database with a corresponding file metadata entry:
     file_metadata_dao = await joint_fixture.container.file_metadata_dao()
-    await file_metadata_dao.insert(EXAMPLE_FILE)
+    await file_metadata_dao.insert(EXAMPLE_METADATA)
 
     # request a stage for the registered file by specifying a wrong checksum:
     file_registry = await joint_fixture.container.file_registry()
     with pytest.raises(FileRegistryPort.ChecksumMissmatchError):
         await file_registry.stage_registered_file(
-            file_id=EXAMPLE_FILE.file_id,
+            file_id=EXAMPLE_METADATA_BASE.file_id,
             decrypted_sha256=(
                 "e6da6d6d05cc057964877aad8a3e9ad712c8abeae279dfa2f89b07eba7ef8abe"
             ),
+            target_object_id=EXAMPLE_METADATA.object_id,
+            target_bucket_id=joint_fixture.staging_bucket,
```

Comment:
This should be the outbox bucket ID instead
registration is from the staging to permanent bucket
staging in this context is from permanent to outbox
Naming might be a bit confusing, as this predates the existence of the staging bucket

Reply:
Fixed in [b0dbdf7](https://github.com/ghga-de/internal-file-registry-service/pull/39/commits/b0dbdf776d534d3ae81d2cd8a5c090afb812690b)